connectivity/suite.go

```diff
@@ -154,7 +184,6 @@ func Run(ctx context.Context, ct *check.ConnectivityTest) error {
 			return check.ResultOK, check.ResultDrop
 		})
 
-	// This policy allows ingress to echo only from client with a label 'other:client'.
```

Comment:
nit: Deleting this line seems unrelated to the rest of the changes? The comment is still valid right?

Reply:
This line is still deleted by this particular commit (37a0224bc257a5b34b329998198eeb400f6d5651).